sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SyntaxTreeExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2019 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace SonarAnalyzer.Helpers
+{
+    public static class SyntaxTreeExtensions
+    {
+        public static Location CreateLocation(this SyntaxTree tree, int start, int end) =>
```

Comment:
It would be better to have the following(s) helper(s):

```csharp
public static Location GetLocation(this SyntaxToken from, SyntaxToken to) =>
	Location.Create(from.SyntaxTree, TextSpan.FromBounds(from.SpanStart, to.Span.End));

public static Location GetLocation(this SyntaxNode from, SyntaxNode to) =>
	Location.Create(from.SyntaxTree, TextSpan.FromBounds(from.SpanStart, to.Span.End));
```